snuba/query/snql/parser.py

```diff
@@ -1537,29 +1543,67 @@ def parse_snql_query(
         with sentry_sdk.start_span(op="processor", description="treeify_conditions"):
             _post_process(query, [_treeify_or_and_conditions], settings)
 
+        # dummy settings and timer dont matter
+        dummy_settings = HTTPQuerySettings()
+        dummy_timer = Timer("snql_pipeline")
```

Comment:
This is not a dummy timer, it's just a timer

Reply:
true... i just didnt think it actually gets used